src/main/java/org/cryptomator/ui/addvaultwizard/CreateNewVaultExpertSettingsController.java

```diff
@@ -92,15 +95,11 @@ public BooleanBinding validShorteningThresholdProperty() {
 	}
 
 	public boolean isValidShorteningThreshold() {
-		try {
-			var value = shorteningThreshold.get();
-			if (value < MIN_SHORTENING_THRESHOLD || value > DEFAULT_SHORTENING_THRESHOLD) {
-				return false;
-			} else {
-				return true;
-			}
-		} catch (NumberFormatException e) {
+		var value = shorteningThreshold.get();
+		if (value < MIN_SHORTENING_THRESHOLD || value > DEFAULT_SHORTENING_THRESHOLD) {
 			return false;
+		} else {
+			return true;
 		}
```

Comment:
```suggestion
		var value = shorteningThreshold.get();
		return value > MIN_SHORTENING_THRESHOLD && value < DEFAULT_SHORTENING_THRESHOLD;
```

Also, I would rename `DEFAULT_SHORTENING_THRESHOLD` to `MAX_SHORTENING_THRESHOLD` and add an alias `DEFAULT_SHORTENING_THRESHOLD = MAX_SHORTENING_THRESHOLD`, if necessary.